rootfs/builder/build.sh

```diff
@@ -157,7 +157,12 @@ if [[ -n "$BUILDPACK_URL" ]]; then
     buildpack_name=$("$buildpack/bin/detect" "$build_root") && selected_buildpack=$buildpack
 else
     for buildpack in "${buildpacks[@]}"; do
-        buildpack_name=$("$buildpack/bin/detect" "$build_root") && selected_buildpack=$buildpack && break
+        shopt -s nocasematch
```

Comment:
This is a good addition. This will make sure `true` and `True` both pass if they are set. :100: